examples/PythonProvider/python-django-webapp-sample/hello_azure/templates/hello_azure/index.html

```diff
@@ -0,0 +1,24 @@
+{% load static %}
+<!doctype html>
+<head>
+    <title>Hello Azure - Python Django Quickstart</title>
+    <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap.min.css' %}">
+    <link rel="icon"href="{% static 'favicon.ico' %}">
+</head>
+<html>
+   <body>
+     <main>
+        <div class="px-4 py-3 my-2 text-center">
+            <img class="d-block mx-auto mb-4" src="{% static 'images/azure-appconfiguration-icon.svg' %}" alt="Azure App Configuration Logo" width="192" height="192"/>
```

Comment:
Let's not open source appconfig icon this way :).

Reply:
This has been updated to use the original Azure logo in the azure example. Thanks!